tests/appsec/contrib_appsec/conftest.py

```diff
@@ -1,30 +1,16 @@
-import unittest.mock
+import ddtrace.auto  # noqa: F401
 
-import pytest
 
-from ddtrace.settings.asm import config as asm_config
-from tests.utils import TracerSpanContainer
-from tests.utils import _build_tree
+# ensure the tracer is loaded and started first for possible iast patching
+print(f"ddtrace version {ddtrace.version.get_version()}")
```

Comment:
This is kind of weird?

Reply:
But it works great.

Reply:
The print is not relevant, it's the import before of course that is.

Reply:
The print line is the line I commented on. Why are we printing something to stdout at module import time without even using a logger?